app/src/main/res/values/colors.xml

```diff
@@ -7,4 +7,5 @@
     <color name="teal_700">#FF018786</color>
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
+    <color name="silver">#E4E4E4</color>
```

Comment:
I will suggest to use more non-specific names of the colors. Instead of using 'silver', 'white', 'black' use names that will say more about the function that color represents - https://www.novatec-gmbh.de/en/blog/name-android-colors-palettes/